spec/injections.spec.js

```diff
@@ -124,6 +95,22 @@ describe("Injections", () => {
       await injections.registerContentScripts();
       expect(spy).toHaveBeenCalled();
     });
+
+    it("fallbacks to contentScripts API on useScriptingAPI set to false", async () => {
+      // Expect Injections to fallback to use contentScripts.register
+      // getBoolPrefSync("useScriptingAPI") returns false.
+      let spyGetBookPrefSync = spyOn(
+        browser.aboutConfigPrefs,
+        "getBoolPrefSync"
+      ).and.returnValue(false);
+
+      let injections = new Injections([buildInjection("desktop")]);
+      expect(spyGetBookPrefSync).toHaveBeenCalledOnceWith("useScriptingAPI");
+      injections.bindAboutCompatBroker(mockBroker);
+      let spyRegister = spyOn(browser.contentScripts, "register");
```

Comment:
Should we also test that there are no scripting API calls when contentScripts are used, and the other way around? In theory there may be duplicate registrations.

Reply:
:+1: That sounds like a good idea, I'm looking into adding that to this test case.

Reply:
I added the additional assertion in [c7a57dd](https://github.com/mozilla-extensions/webcompat-addon/pull/324/commits/c7a57ddad8410aaa1b50a7ccfea78ec7769c727d).